### ISSUE TITLE
Maybe we should add more error codes?

### ISSUE THREAD

**Issue by StefanWallin** [chirag04/react-native-in-app-utils#157]
While digging through my errors I found these two user errors that are not handled by the library. Maybe you want to know about them and incorporate them into the library somehow.

- `ESSSERVERERRORDOMAIN0` - There has been a problem with a family member's previous purchase. (translated from swedish)
- `ESSSERVERERRORDOMAIN5020`
  - NSLocalizedFailureReason: `Have you forgotten your password?`
  - NSLocalizedDescription: `Could not log in`

**Comment by chirag04** [comment]
We don't handle specific error codes anywhere tho. You should see these two error codes in your app directly. no?

**Comment by StefanWallin** [comment]
Yes, you are correct. I think I'm probably thinking more about documentation. It would help a lot I think for newcomers since these error codes almost only appears on google in relation to this library.

Same goes for this the ESKERRORDOMAIN error codes named here: https://github.com/chirag04/react-native-in-app-utils/issues/15#issuecomment-230844435

Would you be interested in establishing a wiki page or part of the readme that mentions those errors? In that case I could make a PR. But only if you think this is a proper place for them.

**Comment by chirag04** [comment]
Yes that’s a good idea. Feel free to send a Pr.
